test/regression_tests.sh

```diff
@@ -126,6 +129,8 @@ sudo rm -rf $ROOT_DIR && sudo mkdir $ROOT_DIR
 sudo chown -R $USER:$USER $ROOT_DIR
 cd $ROOT_DIR
 
+sudo rm -f $TEST_EXECUTION_LOG_FILE $TS_LOG_FILE
```

Comment:
These are test report files that get uploaded to S3 after the test run. Can you please explain why these are being deleted after the test run in AWS Code Build.

**These should NOT be deleted**

Reply:
These files are not deleted after the run instead deleted before a new run to delete logs of previous run . These files should be deleted as these files should have logs only of latest run and not historic runs in case of consecutive runs.